### PR TITLE
Icon updates

### PR DESCRIPTION
**Pull request by mikeNG**

### CODE REVIEW

File: app/src/main/res/values/colors.xml [hunk 1]
```diff
@@ -8,6 +8,7 @@
     <color name="statusBarColor">@*android:color/primary_device_default_settings_light
     </color>
     <color name="red">@*android:color/error_color_device_default_dark</color>
+    <color name="ic_launcher_background">@*android:color/accent_device_default_light</color>
```
Comment (chirayudesai) [inline app/src/main/res/values/colors.xml:11]: Do we want to do this?

**Review by chirayudesai — CHANGES_REQUESTED** [review]
Screenshot_20230310-001550

File: app/src/main/res/values/colors.xml (hunk 1 above)
Comment (theimpulson) [reply app/src/main/res/values/colors.xml:11]: @grote I feel you might be better suited to make the decision here.

**Comment by chirayudesai** [comment]
@mikeNG didn't we have some other issue with this apart from just the color (which is a quick one liner)

**Comment by mikeNG** [comment]
The logo size is different than the original icon, that needs to be fixed

File: app/src/main/res/values/colors.xml (hunk 1 above)
Comment (grote) [reply app/src/main/res/values/colors.xml:11]: I think this way of supplying the resources doesn't work with gradle builds, right?

Comment (theimpulson) [reply app/src/main/res/values/colors.xml:11]: Works fine on both Gradle and AOSP builds

**Review by theimpulson — APPROVED** [review]
Fine by me but I will leave final approval to @grote as the icon color is changed as per the system's accent color and that feels like a major change someone else should approve (in a UI design sense).

**Comment by theimpulson** [comment]
> The logo size is different than the original icon, that needs to be fixed

It's slightly larger compared to the old one but still is fine and follows the adaptive icon guidelines. I will say it's an improvement.

**Review by chirayudesai — APPROVED** [review]

**Comment by chirayudesai** [comment]
(I rebased this to get the new fancy CI to run)

**Comment by chirayudesai** [comment]
Screengrab from emulator (Via CI)

Screenshot_2023-09-26_02-55-16

**Pull request merged**